java/code/src/com/redhat/rhn/manager/kickstart/KickstartScheduleCommand.java

```diff
@@ -1056,6 +1061,10 @@ public ValidatorError validateKickstartPackage() {
             Channel c = i.next();
             serverChannelIds.add(c.getId());
         }
+        if (server.asMinionServer().isPresent()) {
```

Comment:
No, if `cobblerOnly` is true, it does a direct `return null;` and we would not reach this statement.

Reply:
What I meant was that because we are returning null in both cases maybe combining both statements with `||`  should be enough to avoid execution of lines 1057-1063 in case server is the minion.

Reply:
Ahh, you are right. Will push a fix